src/components/Collapsible/Collapsible.tsx

```diff
@@ -66,20 +64,24 @@ const Collapsible: React.FC<CollapsibleProps> = ({
 
   return (
     <Container className={className} withBackground={isOpen}>
-      <Header alignItems="center" onClick={handleHeaderClick}>
-        <StyledIcon
-          isRotated={isOpen}
-          name={SSCIconNames.chevronRight}
-          type={IconTypes.ssc}
-        />
-        <div>
-          <Text size={TextSizes.md}>{title}</Text>
-          <Subject>{subject}</Subject>
-        </div>
+      <Header paddingSize="lg" paddingType="squish" onClick={handleHeaderClick}>
+        <Inline align="center" gap="lg">
+          <StyledIcon
+            isRotated={isOpen}
+            name={SSCIconNames.chevronRight}
+            type={IconTypes.ssc}
+          />
+          <div>
+            <Text size={TextSizes.md}>{title}</Text>
+            <Subject>{subject}</Subject>
+          </div>
+        </Inline>
       </Header>
       {isOpen && (
-        <Content as="div" size={TextSizes.md}>
-          {children}
+        <Content paddingSize="lg" paddingType="square">
```

Comment:
Same here: {SpaceSizes.lg}
And remove paddingType="square" because is the default value for Padbox